pkg/apis/ceph.rook.io/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -139,6 +140,22 @@ func (in *ClusterStatus) DeepCopy() *ClusterStatus {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
@travisn Could it be that you may have forgotten to run `make codegen` in your Ceph dashboard PR?
Seems weird to me that this change occurs after I ran `make codegen`.

Reply:
@galexrt yes i missed it, good thought on the PR checklist

Reply:
@galexrt i should really open a separate PR for it since it's independent of this change

Reply:
@travisn I meant should I add it to the PR template checklist in this PR or #1764?